internal/create/helpers.go

```diff
@@ -0,0 +1,41 @@
+package create
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/gopasspw/gopass/pkg/debug"
+	"github.com/gopasspw/gopass/pkg/fsutil"
+)
+
+func fmtfn(d int, n string, t string) string {
+	strlen := 40 - d
+	// indent - [N] - text (trailing spaces)
+	fmtStr := "%" + strconv.Itoa(d) + "s%s %-" + strconv.Itoa(strlen) + "s"
+	debug.Log("d: %d, n: %q, t: %q, strlen: %d, fmtStr: %q", d, n, t, strlen, fmtStr)
+	return fmt.Sprintf(fmtStr, "", color.GreenString("["+n+"]"), t)
+}
+
+// extractHostname tries to extract the hostname from a URL in a filepath-safe
+// way for use in the name of a secret
+func extractHostname(in string) string {
+	if in == "" {
+		return ""
+	}
+	// help url.Parse by adding a scheme if one is missing. This should still
+	// allow for any scheme, but by default we assume http (only for parsing)
+	urlStr := in
+	if !strings.Contains(urlStr, "://") {
+		urlStr = "http://" + urlStr
```

Comment:
Why not assume `https` by default? Passwords are usually going over TLS secured connections.

Reply:
Because that's the shortest valid scheme. Please note the comment and look at the next few lines: This is really only used to temporarily construct a valid URL. We extract the hostname and throw away everything, i.e. the scheme doesn't escape the method.

Of course we could use https too, but I wouldn't make a difference.